index.html

```diff
@@ -9,18 +9,18 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
-    <div class="mobile-menu">
-      <div class="mobile-menu-close">
-        <i class="mobile-menu-close-btn fas fa-times"></i>
-      </div>
-      <ul class="mobile-menu-options">
-        <li><a href="#projects">Portoflio</a></li>
-        <li><a href="#about-me">About</a></li>
-        <li><a href="#contact">Contact</a>
-        </li>
-      </ul>
-    </div>
     <header>
+      <div class="mobile-menu">
+        <div class="mobile-menu-close">
+          <i class="mobile-menu-close-btn fas fa-times"></i>
+        </div>
+        <ul class="mobile-menu-options">
+          <li><a href="#projects">Portoflio</a></li>
+          <li><a href="#about-me">About</a></li>
+          <li><a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </div>
```

Comment:
Kindly wrap the navigation section with a `nav` tag as advised by the previous reviewer, This helps screen readers to easily navigate through your portfolio website.

[optional] try correcting the typographical errors in your navigation so that it looks professional.